### PR TITLE
BUG - Generated Controller Extensions do not always work with TS

### PR DESCRIPTION
**Pull request by devinea**
#987 BUG - Generated Controller Extensions do not always work with TS

### CODE REVIEW

File: packages/ui5-application-writer/templates/optional/typescript/.babelrc.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
         [
             "transform-ui5",
             {
-                "addControllerStaticPropsToExtend": true
+                "addControllerStaticPropsToExtend": true,
```
Comment (tobiasqueck) [inline packages/ui5-application-writer/templates/optional/typescript/.babelrc.json:7]: `addControllerStaticPropsToExtend` shouldn't be required anymore if we use `overridesToOverride`

**Review by tobiasqueck — CHANGES_REQUESTED** [review]

File: packages/ui5-application-writer/templates/optional/typescript/.babelrc.json (hunk 1 above)
Comment (devinea) [reply packages/ui5-application-writer/templates/optional/typescript/.babelrc.json]: Thanks. Updated in https://github.com/SAP/open-ux-tools/pull/1020/commits/9838a2cd8c9277431da31417abb374eedd3c1355

**Review by tobiasqueck — APPROVED** [review]
* changeset is fine
* only config changes but they are as expected
* didn't test manually but I assume that has been done

**Review by IainSAP — APPROVED** [review]
LGTM

- Updates as expected

**Pull request merged**